crates/debug/tests/dump.rs

```diff
@@ -0,0 +1,30 @@
+use anyhow::{bail, Result};
+use std::env;
+use std::process::Command;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[allow(dead_code)]
+pub enum DwarfDumpSection {
+    DebugInfo,
+    DebugLine,
+}
+
+pub fn get_dwarfdump(obj: &str, section: DwarfDumpSection) -> Result<String> {
```

Comment:
This may be a bit more ergonomic to take `&Path` instead of `&str` for `obj` in the callers?